common/djangoapps/adaptive_learning/migrations/0002_courseadaptivelearningflag.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+import openedx.core.djangoapps.xmodule_django.models
```

Comment:
@tomaszgy If you do 

```python
from openedx.core.djangoapps.xmodule_django.models import CourseKeyField
```

here, you can simplify the definition of the `course_id` field in L24 below.

Reply:
👍